### ISSUE TITLE
Require $system_code variable be set by consumers of this component

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/o-icons#75]
We have a lot of origami-image-service requests with `source=o-icons`, which is actually not useful as we don't know who the real source is. E.G. Is it HTSI/FT/NAR/ScoutAsia?

This is due to the fact that the origami components hardcode a source value in their scss. E.G. https://github.com/Financial-Times/o-icons/blob/master/scss/_mixins.scss#L29

I'm thinking when we introduce `source=` to the Origami-Build-Service, we update the origami components to require users pass in a `$source` or `$system_code` variable in order to use them. We can then use the `$system_code`/`$source` as the value passed to the origami-image-service URLs within the scss.

Waiting on https://github.com/Financial-Times/origami-build-service/issues/67

**Comment by notlee** [comment]
Duplicate of https://github.com/Financial-Times/o-icons/issues/85
